modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/Sidebar.es.js

```diff
@@ -135,7 +135,9 @@ const SidebarTitle = ({className, title}) => (
 	<ClayLayout.ContentRow className={classNames('sidebar-section', className)}>
 		<ClayLayout.ContentCol expand>
 			<div className="component-title">
-				<span className="text-truncate-inline">{title}</span>
+				<h2 className="text-truncate-inline" tabIndex="0">
```

Comment:
This change is responsible for breaking the poshi test path, I already found this path and changed it to expect an h2 tag instead of span.